pandas/tests/extension/category/test_categorical.py

```diff
@@ -154,6 +163,10 @@ class TestMethods(base.BaseMethodsTests):
     def test_value_counts(self, all_data, dropna):
         pass
 
+    @pytest.mark.skip(reason="combine add for categorical not supported")
+    def test_combine_add(self, data_repeated):
+        pass
```

Comment:
does it raise? if so pls tests that

Reply:
This is interesting.  Since `combine` by definition in the docs does an element by element operation, if you do something like:
```
In [1]: import pandas as pd                                                     
                                                                                
In [2]: cat1 = pd.Categorical(values=["one","two","three","three","two","one"], 
   ...:  categories=["one","two","three"], ordered=True)                        
                                                                                
In [3]: s1 = pd.Series(cat1)                                                    
                                                                                
In [4]: s1                                                                      
Out[4]:                                                                         
0      one                                                                      
1      two                                                                      
2    three                                                                      
3    three                                                                      
4      two                                                                      
5      one                                                                      
dtype: category                                                                 
Categories (3, object): [one < two < three]                                     
                                                                                
In [5]: s1.combine("abc", lambda x,y : x+y)                                     
```
In pandas 0.23.0, this fails.  But the binary operation is defined on an element by element basis, so the new implementation will return:
```
Out[4]:
0      oneabc
1      twoabc
2    threeabc
3    threeabc
4      twoabc
5      oneabc
dtype: object
```

IMHO, this is correct, because of what `combine` is supposed to do.  So I will test that.